### PR TITLE
Improve the readme.

### PR DESCRIPTION
**Pull request by lexware**
Added more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +14,12 @@ Gradle is used to handle dependencies.
 - Clone the repo and initialize the [SpongeAPI](https://github.com/SpongePowered/SpongeAPI) dependency: `git clone --recursive https://github.com/LapisBlue/Pore.git`
 - Compile the project using the Gradle wrapper: `./gradlew` (`gradlew` on Windows)
 
+Need Help/Have Questions?
+-------------------------
+
+Having issues using Pore, no problem jump into an IRC client and jump onto #lapis on EsperNet and you'll be sure to get help.
```
Comment (caseif) [inline README.md]: Grammar here is a bit weird

Comment (lexware) [inline README.md]: I have noticed a bit could you point some out, I'm not the best at english.

—

Comment (AesenV) [inline README.md]: There's virtually no pauses; I would write it as such:
```
Having issues with Pore? No problem! Jump onto an IRC client and get on #lapis on EsperNet, and you'll be sure to get help.
```

Comment (lexware) [inline README.md]: Ok i'll change it to that.

**Comment by caseif** [comment]
[Merged!](https://github.com/LapisBlue/Pore/commit/fec09065df82e64cd755fda25133e10ddffe07b5)

**Comment by lexware** [comment]
:)

**Pull request closed without merging**